### PR TITLE
test(integration): add compression tests for c-s

### PR DESCRIPTION
**Pull request by fruch**
since we run into issues c-s around snappy compression we should have a set of tests that can help us identify those kind of issues

### Testing

- [ ]

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [ ] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Comment by mykaul** [comment]
That was a bug/issue in cassandra-stress - shouldn't cassandra-stress CI have such a test? It's not a scylla issue?

File: unit_tests/test_cassandra_stress_thread.py [hunk 1]
```diff
@@ -141,3 +141,33 @@ def cleanup_thread():
 
     assert "latency 99th percentile" in output[0]
     assert float(output[0]["latency 99th percentile"]) > 0
+
+
+@pytest.mark.parametrize("compressor, cql_compression", [
+    ("Deflate", "none"), ("LZ4", "lz4"), ("Snappy", "snappy"), ("Zstd", "none")
```
Comment (mykaul) [inline unit_tests/test_cassandra_stress_thread.py:147]: I didn't know we supported deflate for CQL compression? I think we are mixing sstable compression and CQL protocol level compression?

Comment (CodeLieutenant) [reply unit_tests/test_cassandra_stress_thread.py:147]: Here CQL compression for Deflate and ZStd is none, so there will be no compression in those cases, and the first compression is for Scylla (schema level) and the later one is compression for the protocol, which tipped me off guard when first checking.

There is a `'` after Compressor making it a part of -schema argument

**Review by CodeLieutenant — APPROVED** [review]

**Review by vponomaryov — APPROVED** [review]
LGTM

**Comment by fruch** [comment]
> That was a bug/issue in cassandra-stress - shouldn't cassandra-stress CI have such a test? It's not a scylla issue?

yes, right now we don't have integration test on c-s at all, while we do have them for SCT.

File: unit_tests/test_cassandra_stress_thread.py (hunk 1 above)
Comment (fruch) [reply unit_tests/test_cassandra_stress_thread.py:147]: yes it's mixing it, the issue we had was with cql compression, that only support snappy / lz4

**Pull request merged**